### PR TITLE
Fixup styling and documentation

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
CC @Geod24

### CODE REVIEW

**Comment by AndrejMitrovic** [comment]
Edit: updated with missing `this.`

File: source/agora/node/GossipProtocol.d [hunk 1]
```diff
@@ -16,36 +16,49 @@ module agora.node.GossipProtocol;
...
+/// Ditto
+class GossipProtocol
 {
+    /// Network manager for propagating the received message
     private NetworkManager network;
-    private bool[Hash] receivedMsgCache;
 
-    /// Ctor
+    /// Contains the message cache
+    private bool[Hash] msg_cache;
```
Comment (Geod24) [inline source/agora/node/GossipProtocol.d]: `Set` ?

File: source/agora/node/Network.d [hunk 2]
```diff
@@ -265,11 +265,11 @@ public class NetworkManager
         Sends the message to all the listeners.
 
         Params:
-          msg = the message to send
+            msg = the message to send
```
Comment (Geod24) [inline source/agora/node/Network.d:268]: I think some of my code might suffer from this as well... :D

File: source/agora/node/GossipProtocol.d (hunk 1 above)
Comment (AndrejMitrovic) [reply source/agora/node/GossipProtocol.d]: Yeah I'll fix that up too, one sec.

**Comment by AndrejMitrovic** [comment]
Added another commit.

File: source/agora/node/GossipProtocol.d [hunk 3]
```diff
@@ -14,38 +14,52 @@
 module agora.node.GossipProtocol;
 
 import agora.common.Data;
+import agora.common.Set;
 import agora.node.Network;
 
-/// Procedure of peer-to-peer communication
-class GossipProtocol 
+/// Ditto
+class GossipProtocol
```
Comment (Geod24) [inline source/agora/node/GossipProtocol.d]: 👋

**Comment by AndrejMitrovic** [comment]
Fixed.

**Review by Geod24 — APPROVED** [review]
LGTM, rebase on v0.x.x once the CI is fixed and I'll merge

**Comment by AndrejMitrovic** [comment]
Rebased.

File: source/agora/node/Network.d (hunk 2 above)
Comment (MichaelKim20) [reply source/agora/node/Network.d:268]: I think so too.^-^

**Pull request merged**